AdyenCheckout/AdyenCheckout.swift

```diff
@@ -100,7 +99,7 @@ public class AdyenCheckout: AdyenCheckoutProtocol {
     }
     
     public func createComponent(with paymentMethod: any PaymentMethod) -> AdyenCheckoutComponent? {
-        CheckoutComponentBuilder.build(for: paymentMethod, configuration: configuration)
+        try? CheckoutComponentBuilder.build(for: paymentMethod, configuration: configuration)
```

Comment:
Maybe we should be bold and make this method `throws`.  It will be very helpful for Apple Pay and SDK-based PMs